bigtable_rs/src/bigtable/read_rows.rs

```diff
@@ -33,6 +35,35 @@ pub async fn decode_read_rows_response(
     Ok(rows)
 }
 
+/// Decode the stream of `ReadRowsResponse` into a stream of `Result<(RowKey, Vec<RowCell>)>>`.
+pub async fn decode_read_rows_response_stream<'a>(
+    timeout: &'a Option<Duration>,
+    mut rrr: Streaming<ReadRowsResponse>,
+) -> Pin<Box<dyn Stream<Item = Result<(RowKey, Vec<RowCell>)>> + Send + 'a>> {
```

Comment:
Is there a way we can implement a `RowsStreaam` type to avoid returning a dynamic type? 🤔

Reply:
Yes, this is possible, but I think using `impl Stream` is more idiomatic. It's more flexible too since the caller only cares about the trait impl, not something concrete.

It's a lot simpler now with some improvements too!

Reply:
Looks good now, I think 👍